Fixed Category Creation bug

noglows opened the pull request:
Fixed the category creation bug and added Rspec and Cucumber tests for the bug

noglows committed:
Fixed category creation and added specs

noglows committed:
Small update to Cucumber spec

kariabancroft commented on features/create_category.feature:
```diff
@@ -0,0 +1,15 @@
+
+  Scenario: Successfully go to admin category creation page
+    Given I am on the new admin category page
+    When I fill in "category_name" with "Test Category"
```
What about the other fields?

kariabancroft commented on spec/controllers/admin/categories_controller_spec.rb:
```diff
@@ -34,6 +34,18 @@
+    end
+
+    it 'should render template new' do
+      assert_template 'new'
```
Should there be a specific status returned?

noglows committed:
Updates to the tests for the create category bug fix

noglows commented:
@kariabancroft I updated the tests based on your comments. Added test for the other fields to Cucumber test and added a response status check to the Rspec tests.

kariabancroft commented:
Looks good to close!
